CudaArchitectureSelector.cmake

```diff
@@ -53,6 +53,19 @@
 # specification. Optionally, the list of flags can be filtered by specifying
 # which architectures are not supported by the target.
 #
+# ::
+# 
+#   cas_target_cuda_architectures_plain(
+#    <target>                   # target for which to store the architecures
```

Comment:
Maybe replace `<target>` with `<variable>` instead, since the description says it's just a variable, if I understand it properly?

Reply:
Yes, it should be a variable. I would like to change the name to `cas_variable_cuda_architectures`. Do we need to keep the interface in this repo?

Reply:
I don't think we need to keep the interface here since this is not a part of the `Ginkgo` release. Since this change will not alter the usage of `Ginkgo`, I think it is fine to make that change (since we use a commit hash in the release, we will not alter the behavior there, right?).

Reply:
Yes, I think so, too.